config/webhook/manifests.yaml

```diff
@@ -12,4 +12,61 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
```

Comment:
I think we can remove the tinestanp

Reply:
It's autogenerated, but since we run a script to update the namespace, I can have it remove the null timestamp as well